db/flush_job.cc

```diff
@@ -244,6 +244,7 @@ Status FlushJob::WriteLevel0Table() {
       ThreadStatus::STAGE_FLUSH_WRITE_L0);
   db_mutex_->AssertHeld();
   const uint64_t start_micros = db_options_.env->NowMicros();
+  StopWatch sw(db_options_.env, stats_, FLUSH_TIME);
```

Comment:
how about reusing the time measured for populating `stats.micros`?

Reply:
I could call MeasureTime() and pass the stats.micros value directly. StopWatch allows for disabling statistics collection for a particular type though. Do we want to keep that option?

Reply:
Oh, that's actually some legacy feature in the stats code to decouple internal/external stats and optionally disable internal ones. But it turned out nobody's ever implemented an internal stat so we should probably delete it.

Reply:
Or even if we don't delete it, you can still feel safe calling `measureTime` directly as it does this same check.